example/analysis_options.yaml

```diff
@@ -0,0 +1,9 @@
+include: package:flutter_lints/flutter.yaml
```

Comment:
Had to add also a `analysis_options.yaml` under `example` because VSCode was otherwise complaining, and using the file from the root of the repo